components/bellsound/BellSoundWalkthrough.tsx

```diff
@@ -0,0 +1,119 @@
+import { useState } from "react"
+import { Bike } from "../../lib/bike"
+import SelectFileStep from "./SelectFileStep"
+import UploadStep from "./UploadStep"
+import { Callout, CalloutKind } from "../Callouts"
+import WalkthroughButton from "./WalkthroughButton"
+import dynamic from "next/dynamic"
+
+const ConvertStep = dynamic(() => import("./ConvertStep"), { ssr: false })
+
+export type CommonProps = {
+    onDismiss: () => void
+}
+
+enum WalkthroughStep {
+    SelectFile,
+    Convert,
+    Upload,
+    Done
+}
+
+export default function BellSoundWalkthrough({ bike, onDismiss }: CommonProps & { bike: Bike }) {
+    const [currentStep, setCurrentStep] = useState<WalkthroughStep>(WalkthroughStep.SelectFile)
+    const [selectedFile, setSelectedFile] = useState<File | null>(null)
+    const [convertedFile, setConvertedFile] = useState<Uint8Array | null>(null)
+    const [error, setError] = useState<string | null>(null)
+
+    const onFileSelected = (file: File) => {
+        setError("")
```

Comment:
```suggestion
```

Reply:
The goal of these is to clear the error when the step changes. I just pushed something to make this cleaner